src/views/Pools/svgs/SashVector.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
Pretty sure we have this somewhere already

Reply:
Ah yes indeed, I've pulled in the stuff from ui kit but I think it needs a `z-index` adding, so uikit PR here for that https://github.com/pancakeswap/pancake-toolkit/pull/88